apps/web/lib/org/[orgSlug]/[user]/getServerSideProps.tsx

```diff
@@ -19,7 +19,17 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
     },
   });
   if (team) {
-    return GSSTeamPage({ ...ctx, query: { slug: ctx.query.user } });
+    return GSSTeamPage({
+      ...ctx,
```

Comment:
Don't know why but we are selectively sending query params here, we should make it so that it auto-forwards all params